srv/modules/runners/push.py

```diff
@@ -72,6 +72,16 @@ def proposal(filename = "/srv/pillar/ceph/proposals/policy.cfg", dryrun = False)
     pillar_data.output(common)
     return True
 
+def __create_dirs(path, root):
```

Comment:
I'd be all for that. Just haven't found an answer where such a lib would live. '''/srv/modules/runners''' sounds like a bad idea to me, since salt would pick this up as a runner.